### PR TITLE
k8s continuous deployment + next2 domain

### PR DESCRIPTION
**Pull request by OriHoch**
* kibana on port 80
* add kibana2 and next2 domains for testing the k8s environment
* continuous deployment

### CODE REVIEW

File: .travis.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```
Comment (akariv) [inline .travis.sh]: How about we get this script from a common location instead of copy-pasting it to each repository?
@OriHoch

Comment (OriHoch) [reply .travis.sh]: sure, `curl` it from budgetkey-k8s ?

**Comment by OriHoch** [comment]
@akariv fixed to use script from budgetkey-k8s

**Comment by OriHoch** [comment]
merging without waiting approval because it's blocking other tasks in `budgetkey-k8s`

**Pull request merged**